DataAnalysis.md

```diff
@@ -0,0 +1,36 @@
+# Analysing Model Results
+
+## Analysis Basics
+
+ * Accessing VDI
+ * The different NCI filesystems
+ * Finding data recap
+ * CDO and NCO commands
+ * CLEX Conda environments
+ * Using Jupyter Notebook
+ * Basic Pandas, Xarray and Matplotlib
+
+## Data Formats
+
+ * HDF5/NetCDF - Self-describing data
```

Comment:
Maybe divide into reading and creating. For reading - ncdump.
Should CF standards be included here?